client/src/App.tsx

```diff
@@ -1,21 +1,53 @@
-import React from 'react';
-import logo from './logo.svg';
+import React, { useEffect } from 'react';
 import './App.css';
+import { Provider, useSelector, useDispatch } from 'react-redux';
+import { store } from './store';
+import { selectPrograms } from './selectors';
+import { actionTypes } from './actions/actionTypes';
+
+function FitnessClubApp() {
+  const dispatch = useDispatch();
+  const programs = useSelector(selectPrograms);
+
+  useEffect(() => {
+    // console.log(programs);
+    dispatch({ type: actionTypes.PROGRAMS_FETCH_SUCCEEDED });
```

Comment:
create action
`export const actionGetPrograms = () => ({
  type: actionTypes.PROGRAMS_FETCH_SUCCEEDED,
});
`

and rename action type

Reply:
and when you call this action you are triggered reducer and set undefined in redux